cmd/integration/integration.go

```diff
@@ -855,6 +879,29 @@ func ServeCachedManifestFile() (servingAddress string) {
 }
 
 const (
+	testManifestFile = `kind: Pod
```

Comment:
Yes - it is used by ServeCachedManifestFile.
What I would ideally like to do is to have "two modes" - one testing reading pods from http given by manifest, and the other one reading pods from http given PodSpec. But currently all tests are using the same "http endpoint".